agent/taskresource/fsxwindowsfileserver/fsxwindowsfileserver_windows.go

```diff
@@ -552,18 +552,25 @@ func (fv *FSxWindowsFileServerResource) performHostMount(remotePath string, user
 
 	// New-SmbGlobalMapping cmdlet creates an SMB mapping between the container instance
 	// and SMB share (FSx for Windows File Server file-system)
-	cmd := execCommand("powershell.exe",
+
+	args := []string{
 		"New-SmbGlobalMapping",
 		localPathArg,
 		remotePathArg,
 		creds,
 		"-Persistent $true",
 		"-RequirePrivacy $true",
-		"-ErrorAction Stop")
+		"-ErrorAction Stop",
+	}
+	log_args := make([]string, len(args))
+	log_args = append(log_args, args...)
+	log_args[3] = "<creds>" // Clear the creds for logging.
+	seelog.Debugf("Executing mapping of fsxwindowsfileserver with cmd: %v %v", strings.Join(log_args[:3], " "), strings.Join(log_args[4:], " "))
 
-	_, err = cmd.CombinedOutput()
+	cmd := execCommand("powershell.exe", args...)
+	out, err := cmd.CombinedOutput()
 	if err != nil {
-		seelog.Errorf("Failed to map fsxwindowsfileserver resource on the container instance: %v", err)
+		seelog.Errorf("Failed to map fsxwindowsfileserver resource on the container instance error: %v, out: %v", err, string(out))
```

Comment:
Thanks a lot for this change. This should be very helpful for customers.

Reply:
Np

Reply:
does `out` print creds if its incorrect? just making sure

Reply:
It does not print them in "out" but I added a replace just to be extra cautious incase PS ever changes behavior here.